### DISCUSSION TITLE
Option to add more sections to the home page

### DISCUSSION THREAD

**Discussion by jpalenz77** [pannal/plex-for-kodi#153] [Ideas]
Hello, would be great if PM4K would show ALL the recently added, popular etc... sections the same way you have them configured in plex.

Thanks so much

**Comment by pannal** [comment]
Thanks, I'll investigate this.

**Comment by gyrene2083** [comment]
Just curious is there a way to add this? 
<img width="963" alt="Screenshot 2024-12-18 at 22 03 07" src="https://github.com/user-attachments/assets/2bc46de4-9e62-421c-b164-2fe0508cfd8c" />
The Seasonal Movies. Just curious if it could be done. Thank you.
